app/templates/questionnaire.html

```diff
@@ -7,7 +7,11 @@
 
     <div class="panel panel--error u-mb-s" data-qa="error">
       <div class="panel__header">
-        <h1 class="panel__title venus">This page has {{mapped_errors|length}} errors</h1>
+          {% if mapped_errors|length > 1 %}
+            <h1 class="panel__title venus">This page has {{mapped_errors|length}} errors</h1>
+          {%- else -%}
+            <h1 class="panel__title venus">This page has an error</h1>
+          {%- endif -%}
       </div>
       <div class="panel__body" data-qa="error-body">
         <p class="mars">These <strong>must be corrected</strong> to continue.</p>
```

Comment:
Try this:
{{ ngettext('This', 'These', mapped_errors|length) }} <strong>must be corrected</strong> to continue.